src/main.cpp

```diff
@@ -516,6 +517,10 @@ static void sigaction_signal(int signum, void (*handler)(int)) {
 int main(int argc, char* argv[]) {
     Globals g;
     parse_arguments(argc, argv, g);
+
+    if(!setlocale(LC_CTYPE, "") || !XSupportsLocale()) {
```

Comment:
Nit pick: Can you add a space between if and the opening parentheses?

Reply:
Sure (same issue as for the `fputs`...)

Reply:
Thanks!